metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/hooks/datahub.py

```diff
@@ -303,3 +304,43 @@ def emit(
 
     # Retained for backwards compatibility.
     emit_mces = emit
+
+
+class DatahubCompositeHook(BaseHook):
+    """
+    A hook that can emit metadata to multiple DataHub instances.
+
+    :param datahub_conn_ids: References to the DataHub connections.
+    :type datahub_conn_ids: List[str]
+    """
+
+    hooks: List[DatahubGenericHook] = []
+
+    def __init__(self, datahub_conn_ids: Sequence[str]) -> None:
```

Comment:
sequence is covariant, which isn't really necessary here. a List should be fine

```suggestion
    def __init__(self, datahub_conn_ids: List[str]) -> None:
```